.travis.yml

```diff
@@ -35,10 +40,21 @@ notifications:
     rooms:
       - secure: MkRPjhMDkLMT0tKhSzSAEEdLxKlFrcEw00FBgpcpIl50kSpe3gxCJsm2bz5+LySblCfY1nI5D1+R++eB5cxJzgZvEBWIXQnGJVAV+X7WW1CI1FBZlv6FNLBe5gVPPzo5kIIM1g3o7AaERcraHK9h9S9N8cpnBRHe+Xf6wTRqmwo=
       - secure: iRd6XcgHGFSJ/eS1ZwCUgUVTK0a+JAPmyPEx/OdFbma6DvF8WBhWtW5Q1dyrjQ3JWU8DW2Att9jRnGHd9F5wWWcykxYbD9DXrlfqCZpec9pHzEMAud9MG1PqK0d3y2oyjwps/fQ092PJcn9/y7zX6tzdVWKaKUe4iRDs1u1ehUo=
+install:
+    if [[ "$TRAVIS_BRANCH" == "master" ]] && [[ -n $TRAVIS_TAG ]]; then
```

Comment:
Shouldn't this be the opposite: only do this for tag builds?

Reply:
I believe this is saying that if we are on the master branch and it's a tagged build (-n returns true for non-zero values), then we execute. Are you saying that we should only care if it's a tagged build? (I know that we will be deleting this check, but I just want to make sure I understand what went wrong here) Thanks!

Reply:
Think I had it wrong. But on tagged builds the branch equals the tag: https://docs.travis-ci.com/user/environment-variables/#default-environment-variables